test/test_reductions.py

```diff
@@ -3128,6 +3128,30 @@ def test_tensor_compare_ops_empty(self, device):
             # Check if function raises error on specified zero'd dimension as reduction dim.
             self.assertRaisesRegex(IndexError, "Expected reduction dim", lambda: fn(master_input, dim=1))
 
+    # Tests to ensure that the reduction operators, amax and amix throws errors for zero-dim
+    # tensors (i.e. empty tensors)
+    def test_amax_amin_zero_dim_tensors(self, device):
```

Comment:
Instead of adding a separate test, please add `error_inputs_func` to `amin`, `amax` and `aminmax` in OpInfos in `common_methods_invocations.py`

Reply:
Hi, @ngimel!
I've added ErrorInputs for `amax`, `amin`, and `aminmax`. It was interesting :)

Please take a look and let me know if it is okay. Thanks!

Reply:
You can just add these inputs to SampleInputs in OpInfo tests, and not have a separate test here? SampleInputs will be tested against reference.

Reply:
Sure I've added :)